apps/backend/src/libs/types/repository.type.ts

```diff
@@ -1,7 +1,7 @@
 type Repository<T = unknown> = {
 	create(payload: unknown): Promise<T>;
 	delete(id: number): Promise<boolean>;
-	find(id: number): Promise<null | T>;
+	find(query: T): Promise<null | T>;
```

Comment:
We will use id for find as a common method. If you need anything else this should be a separate method 
```suggestion
	find(id: number): Promise<null | T>;
```

Reply:
This change was suggested [here](https://github.com/BinaryStudioAcademy/bsa-2024-gitfit/pull/38#discussion_r1722141884)

Reply:
I changed this